### ISSUE TITLE
random stop

### ISSUE THREAD

**Issue by joe-bidet** [Scruffydrew/Forza-Horizon-5-Auction-House-Sniper#27]
after around 30 sec it stops working, its opened as admin all settings good, 76 fps

**Comment by matthischober-ui** [comment]
Yeah man same thing i jst click enter every time it stops bc auto clicker doesnt work too
